lib/elements/confirm.js

```diff
@@ -66,8 +69,8 @@ class ConfirmPrompt extends Prompt {
           color.bold(this.msg),
           style.delimiter(this.done),
           this.done
-            ? this.value ? 'yes' : 'no'
-            : color.gray(this.initialValue ? '(Y/n)' : '(y/N)')
+            ? this.value ? yesLabel || 'yes' : noLabel || 'no'
```

Comment:
Do you think it'd be better to move these defaults into the constructor & on the `opts` object directly? In my opinion this is a little hard to read, but certainly doable

Reply:
Agree

Reply:
```js
const { labels } = this.opts;

//

this.done
  ? this.value ? labels.yes : labels.no
  : color.gray(this.initialValue ? labels.defaultYes : labels.defaultNo)
```